### ISSUE TITLE
Large number of ghosted objects causes environment to not update instantly

### ISSUE THREAD

**Issue by consoleSkunk** [ClayHanson/BlocklandRebuilt-IssueBoard#335]
### Bug Description
Updating the environment with a large enough build causes the environment update to be *heavily* delayed.  
Updating the skybox causes the sky to disappear completely until it's ghosted again.

### To Reproduce
1. Load a large build
2. Change environment settings

### Screenshots
https://github.com/ClayHanson/BlocklandRebuilt-IssueBoard/assets/50500602/e5f96190-a6b1-42e2-98a5-af561f2b6b3d

### Metadata:
 - Game Version: `v1.3-rev1783`

**Comment by Potototatoh** [comment]
oxy had a large amount of projectiles on the plains map at one point which caused a large amount of ghosting for a few seconds, afterwards players on his end apparently also wouldn't render properly in 3rd person

tldr: results might vary depending on gpu

**Comment by Zinlock** [comment]
Its not a gpu issue, I believe the ghosting is just really unoptimized for some reason

I noticed having a bunch of vehicles around causes the Bits Received line in the net gui to go way up even when they aren't in use, which can slow ghosting down a lot

**Comment by consoleSkunk** [comment]
Considering this fixed.
